onnxruntime/test/contrib_ops/col2im_test.cc

```diff
@@ -7,8 +7,11 @@
 #include "core/util/math.h"
 
 namespace onnxruntime {
+namespace contrib {
 namespace test {
+using namespace onnxruntime::test;
```

Comment:
Looks like namespace using-directives are forbidden by the style guide: https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
Fairly common on ORT, probably they don't go 100% by Google's guide

````
grep "using namespace onnxruntime::test;" ../../../onnxruntime/ -Rn
onnxruntime/test/contrib_ops/bias_dropout_op_test.cc:37:using namespace onnxruntime::test;
onnxruntime/test/contrib_ops/bitmask_dropout_op_test.cc:27:using namespace onnxruntime::test;
onnxruntime/test/contrib_ops/col2im_test.cc:12:using namespace onnxruntime::test;
onnxruntime/test/contrib_ops/activation_op_test.cc:10:using namespace onnxruntime::test;
onnxruntime/test/contrib_ops/element_wise_ops_test.cc:15:using namespace onnxruntime::test;
onnxruntime/test/contrib_ops/fastgelu_op_test.cc:9:using namespace onnxruntime::test;
onnxruntime/test/contrib_ops/tensor_op_test.cc:10:using namespace onnxruntime::test;
```

Reply:
For my learning, why is the need?

Reply:
FYI, ORT's coding style is based off the Google C++ style guide but has some differences. allowing this can be convenient. in this case, it lets us access names in onnxruntime::test from the onnxruntime::contrib::test namespace.
see the core guidelines links for more info.
https://github.com/microsoft/onnxruntime/blob/d93e6533b7bb9ba32bd5c4f8cc976ec37b838de1/docs/Coding_Conventions_and_Standards.md?plain=1#L60-L63